content/docs/lists-and-keys.md

```diff
@@ -1,115 +1,114 @@
 
-Below, we loop through the `numbers` array using the JavaScript [`map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) function. We return a `<li>` element for each item. Finally, we assign the resulting array of elements to `listItems`:
+A continuació, farem servir el mètode de Javascript [`map()`](https://developer.mozilla.org/ca/docs/Web/JavaScript/Reference/Global_Objects/Array/map) a l'array `numeros`. Aquest ens retornarà un element `<li>` per a cada element de l'array. Després, assignarem l'array resultant a la variable `llistadElements`
 
 ```javascript{2-4}
-const numbers = [1, 2, 3, 4, 5];
-const listItems = numbers.map((number) =>
-  <li>{number}</li>
+const numeros = [1, 2, 3, 4, 5];
+const llistadElements = numeros.map((numero) =>
+  <li>{numero}</li>
 );
 ```
 
-We include the entire `listItems` array inside a `<ul>` element, and [render it to the DOM](/docs/rendering-elements.html#rendering-an-element-into-the-dom):
+Finalment, incloem tota la llista sencera dins un element `<ul>`, i [la renderitzem al DOM](/docs/rendering-elements.html#rendering-an-element-into-the-dom):
 
 ```javascript{2}
 ReactDOM.render(
-  <ul>{listItems}</ul>,
+  <ul>{llistadElements}</ul>,
   document.getElementById('root')
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/GjPyQr?editors=0011)
+[**Prova-ho a CodePen**](https://codepen.io/gaearon/pen/GjPyQr?editors=0011)
 
-This code displays a bullet list of numbers between 1 and 5.
+Aquest codi mostra una llista de números entre l'1 i el 5.
 
-### Basic List Component {#basic-list-component}
+### Component de llista bàsic {#basic-list-component}
 
-Usually you would render lists inside a [component](/docs/components-and-props.html).
+Normalment renderitzaràs les llistes dins d'un [component](/docs/components-and-props.html).
 
-We can refactor the previous example into a component that accepts an array of `numbers` and outputs a list of elements.
+Podem refer l'exemple anterior fent un component que accepti una array de `numeros` i generi una llista d'elements.
 
 ```javascript{3-5,7,13}
-function NumberList(props) {
-  const numbers = props.numbers;
-  const listItems = numbers.map((number) =>
-    <li>{number}</li>
+function LlistaNumeros(props) {
+  const numeros = props.numeros;
+  const llistadElements = numeros.map((numero) =>
+    <li>{numero}</li>
   );
   return (
-    <ul>{listItems}</ul>
+    <ul>{llistadElements}</ul>
   );
 }
 
-const numbers = [1, 2, 3, 4, 5];
+const numeros = [1, 2, 3, 4, 5];
 ReactDOM.render(
-  <NumberList numbers={numbers} />,
+  <LlistaNumeros numeros={numeros} />,
   document.getElementById('root')
 );
 ```
+Quan executis aquest codi rebràs un avís que diu que s'ha de donar una clau(`key`) a cada elements de la llista. Una clau (`key`) és un atribut de cadena especial que has d'incloure en crear llistes d'elements. Parlarem de per què és important a la següent secció.
 
```

Comment:
I would suggest keeping the same line breaks as in the original English version. Like this, line 72 should be a line with no content but just a line break.

Reply:
it was not my intention to change it. I'm sorry.